src/components/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {Nav, NavItem, NavLink} from 'reactstrap';
+
+
+const Footer = () => {
+  return (
+    <>
+    <Nav>  
+        <NavItem>
+          <NavLink
+            active
+            href="#"
+          >
+            Link
+          </NavLink>
+        </NavItem>
+        <NavItem>
+          <NavLink href="#">
+            Another Link
+          </NavLink>
+        </NavItem>
+      </Nav>
```

Comment:
I would recommend updating this with actual links and not just filler text.  Oftentimes, footers don't need a lot of content.  It can just be the copyright with your names on it.